src/tasks/initiate_new_auction.ts

```diff
@@ -85,6 +85,13 @@ const initiateAuction: () => void = () => {
 
       console.log("Using EasyAuction deployed to:", easyAuction.address);
 
+      const balance = await auctioningToken.callStatic.balanceOf(
+        caller.address,
+      );
+      if (sellAmountsInAtoms.gt(balance)) {
```

Comment:
I suppose also equality should work?
```suggestion
      if (sellAmountsInAtoms.gte(balance)) {
```

Reply:
I am confused. For equality we don't wanna throw an error, right?

Reply:
... yes, of course!